src/entry.js

```diff
@@ -1 +1,3 @@
-module.exports = require('./operations')
+import operations from './operations'
+
+module.exports = operations
```

Comment:
I would prefer either

- deleting this file entirely and changing `main` in `package.json` to point to `src/operations/index.js`
- copying the contents of `src/operations/index.js` into this file, and deleting `src/operations/index.js`

I'd lean a bit more towards the latter, myself.